SECURITY.md

```diff
@@ -0,0 +1,5 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+Please report security issues to <email>
```

Comment:
Obviously this PR can't be accepted as-is because that's not a valid email address.